src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -224,10 +231,26 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	public toggleEditMode(editMode?: boolean): void {
 		this.isEditMode = editMode !== undefined ? editMode : !this.isEditMode;
 		this.cellModel.isEditMode = this.isEditMode;
+		if (this.isEditMode === false) {
```

Comment:
Typically for boolean checks it's fine to just do `!this.isEditMode` instead of specifically checking for false. The only reason you'd want to do that is if you think it could be undefined and want to make sure you're specifically only allowing false values.